collect_app/src/main/res/values/strings.xml

```diff
@@ -150,11 +147,6 @@
     <string name="submission_url">Submission path</string>
     <string name="font_size">Text font size</string>
     <string name="change_font_size">Text font size</string>
-    <string name="access_error">Error while accessing %s: </string>
-    <string name="manifest_server_error">Manifest reply does not report an OpenRosa version — bad server?</string>
-    <string name="root_element_error">Root element is not &lt;manifest\&gt; — was %s</string>
-    <string name="root_namespace_error">Root element Namespace is incorrect: %s</string>
-    <string name="manifest_tag_error">Manifest entry %s is missing one or more tags: filename, hash, or downloadUrl</string>
     <string name="form_download_progress">%1$s. Getting media files: %2$s of %3$s</string>
     <string name="fetching_manifest">%s. Getting manifest </string>
```

Comment:
This one can also be rolled into a generic error, I think.

Reply:
I'm pretty sure this string is just used for progress updates. Not sure what you mean.

Reply:
More of a note to self to check how it’s used. The user should not see the word manifest anywhere.

Reply:
Yeah I think a lot of the progress updates in the app are things we could get rid of now. A lot of detail that doesn't help anyone!